server/meca/Readme.md

```diff
@@ -6,10 +6,13 @@ xPub component to generate and export a Manuscript Exchange Common Approach (MEC
 
     const {mecaExport} = require('@elifesciences/xpub-meca-export')
 
-    mecaExport(manuscriptObject, manuscriptContent, clientIp)
+    mecaExport(manuscriptObject, getContentFunction, clientIp)
         .then(() => console.log('Successfully uploaded archive to SFTP'))
         .catch(err => console.error(err))
 
+The `getContentFunction` has the signiture of one argument, the `File` object which
+will return the file contents of that object.
+
```

Comment:
I'm not a huge fan of this interface, as it feels like we're shoehorning in some functionality so the export works.

We could change MecaExport to a class, and do the following:
```
const meca = new Meca({
  getContent: S3Storage.getContent
})
meca.export(manuscript, clientIp)
```

It's also easier to add functionality to a class in the future if we need to fetch content from other places, but more than anything, it's a lot easier to read IMO

Reply:
This would be my preference too ... just trying to make the least risky change at this stage of the project.

Reply:
Happy for this to go in a separate issue?

Reply:
Yeah ... I'm seeing this as part of the wider architecture issue. Although the MECA component is planned on being adopted by `pubsweet` ... that is another reason I've chosen to keep it as it is at the moment. But I'm happy this goes in as an issue on its own.